src/test/java/com/at/globalclasses/MongoDBUtils.java

```diff
@@ -22,6 +23,16 @@ public static String getRandomID(String env, String mDataBase, String collection
         return id;
     }
 
+
+    public static String compareID(String env, String mDataBase, String collection, String response) {
+        MongoDBConnection db = new MongoDBConnection(env, mDataBase);
+
```

Comment:
Here you can extract the id string value before to call getObjectById. Also this method is not compating anything at all so could be renamed.

Reply:
I changed the name and i extracted the id from here and i send the id value to getObjectById